twingate/client_connector.go

```diff
@@ -51,6 +52,29 @@ func (client *Client) createConnector(remoteNetworkID string) (*Connector, error
 	return &connector, nil
 }
 
+func (client *Client) readTestConnectors() ([]string, error) { //nolint
```

Comment:
I think it is better just to have a `readConnectors` method and do the filtering on the caller `testSweepTwingateConnector` (same comment of other clients).

Reply:
Changed to common `readConnectors`: 6944e77ae8d0ef6510cc7fba00fd165b44b17d65